bench-tps/.gitignore

```diff
@@ -0,0 +1,3 @@
+/target/
+/config/
```

Comment:
the /config/ and /config-local/ here shouldn't be needed anymore since we always now use solana_root in the multinode-demo/ scripts

Reply:
Using solana_root is what leads to /config/ and /config-local/ being created, no?

Reply:
Yeah but not in the bench-tps/ subdirectory